notes/stv/20200408-Spark-on-K8s.txt

```diff
@@ -461,17 +463,17 @@ git clone https://github.com/devshlabs/spark-kubernetes
 # In this example, we set it to stvoutsin/spark:2.4.4
 #[user@@k8s-master]
 
-nano kubernetes/spark-worker-controller.yaml
+nano spark-kubernetes/kubernetes/spark-worker-controller.yaml
 ..
 
     spec:
       containers:
         - name: spark-worker
-          image: stvoutsin/spark:2.4.4
+       stvoutsin/spark:2.4.4   image: 
```

Comment:
Is this a typo ?

    image: stvoutsin/spark:2.4.4

makes more sense than

    stvoutsin/spark:2.4.4   image:

Reply:
Yep thats a typo, should be -> image: stvoutsin/spark:2.4.4